doc/Caliptra.md

```diff
@@ -189,7 +187,7 @@ In this version, Caliptra Silicon RoT desired capabilities address the basics of
 
 ### Supply Chain Security
 
-* **Mutable Code Integrity:** The objective here is to prove the device is running genuine firmware that the device manufacturer can vouch for its authenticity and integrity, and the device owner can ensure only authorized updates are applied to the device. This flow is aligned with [Reference 10](#ref-10) and can be achieved with dual signature verification of equal imposition.
+* **Mutable Code Integrity:** The objective here is to prove the device is running genuine firmware that the device manufacturer can vouch for its authenticity and integrity, and the device owner can ensure only authorized updates are applied to the device. This flow is aligned with [Reference 9](#ref-9) and can be achieved with dual signature verification of equal imposition.
```

Comment:
Not necessarily something to fix in this PR but I'm not sure I follow what "equal imposition" means.